api/src/integration/resources/schema.sql

```diff
@@ -0,0 +1,16 @@
+create table config (
```

Comment:
Add comment describing how this gets used / where it's referenced from?

Reply:
Done.